packages/styles/src/components/form-check.scss

```diff
@@ -163,6 +180,16 @@
 
   &:hover > .form-check-input:not([disabled]) {
     background-color: rgba(var(--post-contrast-color-rgb), 0.1);
+
+    @include utility-mx.high-contrast-mode {
+      border-color: Highlight;
+
+      &:checked,
+      &::after,
+      &:checked::after {
+        background-color: Highlight;
```

Comment:
Maybe we can use `AccentColor` and as a fallback Highlight. I'm not sure of what I'm saying, but it looks like it's the color to show that an element is active. It's supported by Firefox (though I don't see the difference) and under a flag on blink engine browsers.